### ISSUE TITLE
OSC (open sound control) - Solo button in mixer doesn't respond to OSC message.

### ISSUE THREAD

**Issue by gitsmol** [surge-synthesizer/surge#7417]
**Bug Description:**
The solo button in the scene mixer doesn't respond to OSC messages.

**Surge XT Version**
Version: 1.3.0.20a8e16
Plugin Type: Standalone
Bitness: 64-bit

**Reproduction Steps:**
Send an OSC message to '/param/a/mixer/osc1/solo' with value 'True' or '1.0' (float). See also this python script:
```python
from osc4py3.as_eventloop import *
from osc4py3 import oscbuildparse as obp
from osc4py3.oscmethod import *

def set_mute():
    msg = obp.OSCMessage("/param/a/mixer/osc1/solo", ",f", [False])
    osc_send(msg, "oscout")
    osc_process()

### config
ip = "127.0.0.1"
surgeOSCInPort = 53210	#Surge XT default OSC in port

osc_startup()
osc_udp_client(ip, surgeOSCInPort, "oscout")

set_mute()

osc_terminate()

```

**Expected Behavior:**
The selected channel is solo'ed.

**Computer Information (please complete the following!):**
- OS: Mac OS 13.6.1
- Host: Standalone

**Additional Information:**
The solo button does correctly send its value over OSC when it is changed. The mute button works fine.

**Comment by baconpaul** [comment]
Does it solo and the buttons are wrong or does it not solo?

**Comment by gitsmol** [comment]
Good question. It's only the button that doesn't light up; I tested with the noise generator and it does solo the channel, just doesn't show it.

**Comment by baconpaul** [comment]
Great then yeah I have a pretty good idea what this is. Thanks!

**Comment by baconpaul** [comment]
and indeed that's what it was. Fixed in #7438 and a modified version of your script added to the repo. thanks!
